### DISCUSSION TITLE
Saleor v3.0 release date; recommended version to use at the moment

### DISCUSSION THREAD

**Discussion by satvikt** [saleor/saleor#9019] [General]
Looking at the saleor [Roadmap & Releases](https://github.com/saleor/saleor/projects/12), we can see that Saleor v3.0 was expected to release in Q4 2021. However the [tags tab](https://github.com/saleor/saleor/tags) does not refer to an official stable 3.0 release. 

Could we get a tentative timeline as to when a stable version of Saleor 3.0 will be officially released?

Also, We are developing an ecommerce store based on Saleor, and have tentatively planned a release within 2 months. What do you recommend we start using - Saleor v3.0 or v3.1?

**Comment by karolkielecki** [comment]
Hi @satvikt,

Thanks for your question. We plan to release a release candidate for 3.0 in the next weeks but can't promise when exactly. We'll update the roadmap soon.

We recently discussed which option would be the best for new projects within the team.  Actually, for new projects planned to release within 2 months, we recommend using 3.1-- it introduces Gift Cards and other improvements.

**Reply by iba-1** [reply]
Hey @karolkielecki,
is there a usable and compatible frontend version?
Can we use [saleor-storefront@3.0.0-b.2](https://github.com/saleor/saleor-storefront/tree/3.0.0-b.2) with saleor (core) v3.1?

Should we wait for an initial release of react-storefront? It doesn't seem "complete" at the moment, and it would be a bit of a pain customizing and then fetching upstream to include any further development from your side.

We don't mind using an unstable version but we would love to know which version of saleor is compatible with which version of saleor-storefront/react-storefront and dashboard, and what will we lose by using either version!

_If it sounds a bit passive-aggressive is due to my poor english writing skill, we're really glad you brought this platform out in OSS and we've been working with it since 2.4_ 🤪

**Reply by karolkielecki** [reply]
Well, saleor-storefront is not maintained anymore but you can still use it. As you mentioned, we're also working on react-storefront and it will replace saleor-storefront. It's a starter kit and consider it as an alpha stage. Personally, I would go with react-storefront. Could you please tell us more about react-storefront? What do you expect to have from it?

**Reply by iba-1** [reply]
One thing that may trouble us as is that no other PG has been implemented other than Dummy and Basic Stripe (will 3dsv1 and V2 be supported?), as we had to implement support for 3ds v1-v2 both on saleor-storefront@2.11 and saleor-core and I recall it as a not so straightforward task.

Nevertheless, we'll kick off with both and see which one performs better both by overall and dev experience.

Edit: Can we use saleor-storefront@3.0.0-b.2 with saleor (core) v3.1 or is there any incompatibility issue?

**Reply by karolkielecki** [reply]
IMO, there shouldn't be any issues if you use saleor-storefront b.2 + Saleor core v3.1.
